downstream/modules/automation-hub/proc-adding-containers-remotely-to-the-automation-hub.adoc

```diff
@@ -0,0 +1,33 @@
+//Module included in the following assemblies:
+
+[id="adding-containers-remotely-to-the-automation-hub"]
+
+= Adding containers remotely to the automation hub
+
+There are two ways to add containers to the automation hub:
+
+* Create Remotes
+* Execution Environment
+
+.Procedure
+To add a remote registry:
+
+. In automation hub, click on *Execution Environments* in the left menu
+pane. Two choices will become available, *Execution Environments*, and *Remote
+Registries*.
+
```

Comment:
Suggested change: Remove on from "click on" in the first 3 bullets.

Also rephrase to remove "left" for accessibility compliance.